### DISCUSSION TITLE
Too many targets

### DISCUSSION THREAD

**Discussion by superzanti** [ggandor/leap.nvim#236] [General]
Does anyone know if there is a way to increase the number of phases if there are too many targets?
I see a green highlight and a purple highlight, but I can't press the keys in the purple highlights to continue navigating.

**Comment by ggandor** [comment]
https://github.com/ggandor/leap.nvim/blob/eca8108dcd5f41fbde2a674dceb58b81ed887dc8/doc/leap.txt#L48-L51

https://github.com/ggandor/leap.nvim/blob/eca8108dcd5f41fbde2a674dceb58b81ed887dc8/doc/leap.txt#L329-L333

**Comment by superzanti** [comment]
Ah Thank you!

Sorry I missed this in the documentation.
